components/typography/typography.js

```diff
@@ -1,7 +1,5 @@
 import '@webcomponents/shadycss/custom-style-interface.min.js';
 
-/* TODO: deal with the mixins. */
```

Comment:
This file sets up the fonts and scoped d2l-typography selectors as was done previously.  The expectation is that this gets included on the page once, and it sets up the typography at the document level.